module/VuFind/src/VuFind/Search/Base/Params.php

```diff
@@ -1331,11 +1331,12 @@ protected function initRangeFilters($request)
      * Support method for initDateFilters() -- normalize a year for use in a
      * year-based date range.
      *
-     * @param ?string $year Value to check for valid year.
+     * @param ?string $year     Value to check for valid year.
+     * @param bool    $rangeEnd Is this the end of a range?
      *
      * @return string      Formatted year.
      */
-    protected function formatYearForDateRange($year)
+    protected function formatYearForDateRange($year, $rangeEnd = false)
```

Comment:
This argument is unused; I understand why it is here, but we should probably add appropriate annotation so it doesn't trigger static analysis warnings.

Reply:
Annotations added.